tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -449,6 +449,15 @@ spec:
 			stdOut = helper.CmdShouldFail("odo", "link", "EtcdCluster/example")
 			Expect(stdOut).To(ContainSubstring("already linked with the service"))
 
+			// Before running "odo unlink" checks, wait for the pod to come up from "odo push" done after "odo link"
+			pods = oc.GetAllPodsInNs(project)
+			componentPod := regexp.MustCompile(fmt.Sprintf(`%s-.[a-z0-9]*-.[a-z0-9]*`, componentName)).FindString(pods)
+			ocArgs = []string{"get", "pods", componentPod, "-o", "template=\"{{.status.phase}}\"", "-n", project}
+			helper.WaitForCmdOut("oc", ocArgs, 1, true, func(output string) bool {
+				return strings.Contains(output, "Running")
+			})
+
+			// stdOut = helper.CmdShouldRunWithTimeout(1*time.Minute, "odo", "unlink", "EtcdCluster/example")
```

Comment:
commented code found

Reply:
please remove this commented code @dharmit , else all looks good